CHANGELOG.next.asciidoc

```diff
@@ -140,6 +140,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add DeDot for kubernetes labels and annotations. {issue}9860[9860] {pull}9939[9939]
 - Add more meaningful metrics to 'performance' Metricset on 'MSSQL' module {pull}10011[10011]
 - Rename some fields in `performance` Metricset on MSSQL module to match the updated documentation from Microsoft {pull}10074[10074]
+- Rename 'db' Metricset to 'log' in MSSQL Metricbeat module {pull}10109[10109]
```

Comment:
I have actually been thinking about this. This were my thoughts:
- `transaction_log`: It might confuse user thinking that it's some metric related to transactions but it's not. It's the internal log of data and operations that must be processed and flushed to _data_ files. Here's a nice explanation https://www.c-sharpcorner.com/article/learn-about-sql-server-architecture/
- `data_log`: Means nothing and everything
- `temporal_log`: I thought "maybe" but it's actually giving lot of meaning to something that maybe it's not correctly described as "temporal".
- `log`: Well... they are not "Da Logs" in the Elasticsearch term but it should not be not possible to confuse

Reply:
I have just discover that Microsoft also call it "transaction log" even when it's the log for everything.

https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-log-space-usage-transact-sql?view=sql-server-2017

Reply:
@jsoriano maybe you can check the names mentioned above? @ruflin do you have any idea too?

Reply:
I'd go for `transactionlog` (without underscore), specially if Microsoft also calls it this way. They also mention that this is about transactions: https://docs.microsoft.com/en-us/sql/relational-databases/logs/the-transaction-log-sql-server?view=sql-server-2017

> this is not Filebeat but Metricbeat.

The separation between them can be removed in the future.

Reply:
+1. With and without _ works for me.

Reply:
Change to `transaction_log`. I have used underscore instead of CamelCase because this way field names on JSON also follows naming conventions instead of having `transactionLog` and `transaction_log` mixed in the module.

Reply:
Well, the alternative was not camel case but `transactionlog`. But it's true that we already have other metricsets with underscore, so as you prefer, both `transactionlog` and `transaction_log` work for me.

Reply:
as windows uses 2 words, I would go with _